app-android/src/main/res/drawable-night-v31/splash_branding_image.xml

```diff
@@ -0,0 +1,48 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="200dp"
+    android:height="80dp"
+    android:viewportWidth="200"
+    android:viewportHeight="80">
```

Comment:
The size of the images in Figma did not meet the requirements if they were imported as is, so the size was manually corrected.

https://developer.android.com/develop/ui/views/launch/splash-screen#dimensions
> Branded image: this must be 200×80 dp.